roles/ceph-common/tasks/main.yml

```diff
@@ -11,6 +11,23 @@
        cache_valid_time=3600
   with_items: ceph.debian_pkgs
 
+- name: create ceph log directory
+  file: dest=/var/log/ceph
+        state=directory
+        owner=root
+        group=adm
+        mode=2750
+
+- name: Change ceph log directory permissions with acl
```

Comment:
is this `acl` task necessary given that we're setting the setgid bit on the directory which should ensure new files get created with the same 'adm' group ?